keyboards/keychron/q5/info.json

```
@@ -0,0 +1,51 @@
+{
+    "manufacturer": "Keychron",
+    "keyboard_name": "Keychron Q5",
+    "maintainer": "lalalademaxiya1",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "ROW2COL",
+    "eeprom": {
+        "backing_size": 4096
```

Config here is invalid
```suggestion
        "wear_leveling": {
            "backing_size": 4096
        }
```